corehq/apps/widget/static/widget/js/testUDPNetwork.js

```diff
@@ -0,0 +1,209 @@
+/**
+Based on AWS boilerplate

```

Comment:
Should this file be committed to production? Looks like it's only referenced in `checkNetwork` which is never called. This seems like it would be useful to have as documentation, but not especially useful checked in.